app/upgrades/v20/upgrades.go

```diff
@@ -645,3 +696,62 @@ func MigrateChangeRewardDenomsProposal(
 	}
 	return nil
 }
+
+// SetICSConsumerMetadata sets the metadata for launched consumer chains
+func SetICSConsumerMetadata(ctx sdk.Context, providerKeeper providerkeeper.Keeper) error {
+	for _, consumerID := range providerKeeper.GetAllActiveConsumerIds(ctx) {
```

Comment:
We have `GetAllLaunchedConsumerIds` so we can use that one directly (?)
s/consumerID/consumerId.

Reply:
> s/consumerID/consumerId.

The linter doesn't like consumerId for some reason.

Reply:
> We have GetAllLaunchedConsumerIds so we can use that one directly (?)

It doesn't really matter. Leave it like this. Before the v20 upgrade, the number of consumer IDs is low. :)